### PR TITLE
Fix typo from a rename

### PR DESCRIPTION
**Pull request by e-dard**
Made a typo before Christmas when renaming for `golint`.

/cc @jsternberg

### CODE REVIEW

**Comment by jwilder** [comment]
:+1:

File: services/graphite/errors.go [hunk 1]
```diff
@@ -4,11 +4,11 @@ import "fmt"
 
 // An UnsupposedValueError is returned when a parsed value is not
```
Comment (bbinet) [inline services/graphite/errors.go]: you should update the comment too

**Comment by corylanou** [comment]
+1

**Comment by jsternberg** [comment]
:+1:

**Pull request merged**